rskj-core/src/main/java/co/rsk/mine/gas/StableMinGasPriceProviderFactory.java

```diff
@@ -0,0 +1,43 @@
+package co.rsk.mine.gas;
+
+import co.rsk.config.mining.HttpGetStableMinGasSystemConfig;
+import co.rsk.config.mining.StableMinGasPriceSystemConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class StableMinGasPriceProviderFactory {
+
+    private static final Logger logger = LoggerFactory.getLogger("StableMinGasPrice");
+
+    private StableMinGasPriceProviderFactory() {}
+
+    public static MinGasPriceProvider create(StableMinGasPriceSystemConfig config, DefaultMinGasPriceProvider fallBackProvider) {
+        MinGasPriceProviderType method = config.getMethod();
+        if (method == null) {
```

Comment:
Here it's a point that could be improved if we use the enum as I suggested, it would return the default always when the method isn't of the type `HTTP_GET` and `ETH_CALL`.

Reply:
This should never happen as an error is thrown by the config class but I wanted to cover this edge case if the feature is modified in the future.